core/store/src/columns.rs

```diff
@@ -237,6 +237,13 @@ pub enum DBCol {
 }
 
 impl DBCol {
+    /// Whethere this column can be written to only once.
```

Comment:
to be correct: it doesn't mean that it can be written 'only once' - but we should rather say that we would store only '1 value' in  there (not sure how to phrase it ;-)